### ISSUE TITLE
ADSBx to VRS

### ISSUE THREAD

**Issue by countryboy057** [kenigbolo/adsbexchange-js-utility#45]
Is their a way with your utility to pipe this data into my local Virtual Radar install. ADSBx use to allow feeders a TCP stream of data so we can host it on our own private VRS install. When they shut this off I was told that
"There's the REST API. That is the only way this will work going forward.

A simple golang app that makes REST API requests to ADSBx that converts the REST API data to VRS using Aircraft JSON is the solution. I simply don't have time to write it."

How hard would this be to do ? I know alot of people like to have their own local VRS install to watch instead of the ADSBx webpage.

Thanks
Branden

**Comment by kenigbolo** [comment]
@countryboy057 I, unfortunately, do not have a lot of time over the holidays however I could attempt to add a data converting module to this utility. The data converting module would transform the API response received from ADSBx into Aircraft JSON using the specs outlined at http://www.virtualradarserver.co.uk/documentation/formats/AircraftList.aspx#response

I would not have the time to do this right away but could get back to it after the holidays. Would that work for you?

**Comment by countryboy057** [comment]
Absolutely. It was not anything that was immediate at all. Please enjoy the holiday and time with your family! Happy New year !

**Comment by countryboy057** [comment]
Were you ever able to look into this?

**Comment by kenigbolo** [comment]
@countryboy057 Unfortunately didn't have the time yet. I still have this in my backlog so for sure I'll come back to it soon
